pkgs/by-name/dp/dprint/plugins/default.nix

```
@@ -45,5 +53,11 @@ let
     name: _:
     nameValuePair (removeSuffix ".nix" name) (import (./. + "/${name}") { inherit mkDprintPlugin; })
   ) files;
+  # cb arg: can pass a callback to filter out plugins from nixpkgs or a list of plugins
+  getPluginList =
+    cb:
+    map (p: "${builtins.toString p}/plugin.wasm") (
+      builtins.filter isDerivation (if builtins.isFunction cb then cb plugins else cb)
+    );
```

I think this can work:

```suggestion
  getPluginList = cb: map (p: "${p}/plugin.wasm") cb plugins;
```